apps/server/src/modules/task-card/uc/task-card.uc.ts

```diff
@@ -114,16 +114,13 @@ export class TaskCardUc {
 		card.title = params.title;
 		card.course = course;
 		card.dueDate = params.dueDate;
+		card.visibleAtDate = params.visibleAtDate;
```

Comment:
if it's not set the update doesn't work correctly. There has to be the visibleAtDate-property declared and set to undefined. Otherwise it will not be deleted.